src/main/java/org/jenkinsci/plugins/workflow/actions/ErrorAction.java

```diff
@@ -26,9 +26,11 @@
 
 import groovy.lang.GroovyClassLoader;
 import groovy.lang.MissingMethodException;
+import groovy.lang.MissingPropertyException;
 import hudson.remoting.ProxyException;
 import javax.annotation.CheckForNull;
 import org.codehaus.groovy.control.MultipleCompilationErrorsException;
+import org.codehaus.groovy.tools.GroovyClass;
```

Comment:
Btw, is this actually used? If not, would be nice to remove it.

Reply:
No it is it not needed, I blame my IDE, but in the end I'm the one controlling it so I should probably blame myself ;) Fixed now anyways.